pipelines/enterprise-contract.yaml

```diff
@@ -120,7 +120,7 @@ spec:
         resolver: bundles
         params:
           - name: bundle
-            value: quay.io/enterprise-contract/tekton-task:latest@sha256:88755b87abb38842824ed2489e7bb78d3d6fb9179ee046ee6e2542e50a5e24e0
+            value: quay.io/conforma/tekton-task:latest@sha256:88755b87abb38842824ed2489e7bb78d3d6fb9179ee046ee6e2542e50a5e24e0
```

Comment:
I think this is the digest `2d5d51c5402cae2d521affbb280fcd00d2ea39fe63be80375f2663afe6c30e96`

Reply:
This digest bump happened in recently merged #3046 , so after the rebase, this is back to a repo change only.